SmartPlaces.Facilities/lib/IngestionManager.Mapped/src/MappedGraphIngestionProcessor.cs

```diff
@@ -252,8 +275,9 @@ private async Task GetPointsAsync(IDictionary<string, BasicDigitalTwin> twins, I
                                     if (pointElement.TryGetProperty("exactType", out var pointExactType))
                                     {
                                         AddTwin(twins, pointElement, pointDtId, pointExactType.ToString());
+                                        var relationshipProperties = new Dictionary<string, object>();
```

Comment:
Any reason for assigning this to variable every nested loop instead of inline?

Reply:
Want to make sure that the properties for each relationship never get accidentally assigned to the wrong relationship. If we do eventually add properties to relationships other than "isfedby" want to keep the code intent clear.